packages/docs/components/TabWrapper/TabWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useState } from 'react';
+
+interface Context {
+  activeTab: string;
+  setActiveTab(activeTab: string): void;
+}
+
+export const ActiveTabContext = createContext<Context>({
+  activeTab: '',
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  setActiveTab: () => {},
+});
+
+export const TabWrapper: React.FC = (props) => {
+  const [openTab, setOpenTab] = useState('examples');
+  const setTab = (newTab) => setOpenTab(newTab);
+
+  return (
+    <ActiveTabContext.Provider value={{ activeTab: openTab, setActiveTab: setTab }}>
+      {props.children}
+    </ActiveTabContext.Provider>
+  );
+};
```

Comment:
Let's try and modify this component and use it instead of `useContext` + `Tabs`. Context is probably not a good use case for this component since state doesn't need to be shared between multiple components: 
1. Rename this to `TabNavigation` and just use local state.
2. Make this config-based:
```tsx
<TabNavigation
  tabs={[{ id: 'examples', title: 'Examples', render: () => (...) }]}
/>
```
3. Use and update query params in the URL to keep consistency for routing in and out of pages.

Number 2. is very import if we decided to split the tabs into new files:
```tsx
import { TableExamples } from './TableExamples';
import { TableGuidelines } from './TableGuidelines';

// ...

<TabNavigation
  tabs={[
    { id: 'examples', render: TableExamples, ... },
    { id: 'guidelines', render: TableGuidelines, ... },
    // ...
  ]}
/>
```

Reply:
Agree with this approach

Reply:
Do I need to do anything to keep the query param consistent while moving between pages?

Reply:
After thinking a little about it, we either can go the query param route, which may be tricky to implement perfectly. Or we can just read and write to local/session storage which might be the better route 🤔

Reply:
Wouldn't it be better to use a Context than writing into the local session? What's the concern about the Context approach?

Re… query param I'm out of depth on that one considering how Next.js handles routing.